src/main/resources/plugin.yml

```diff
@@ -0,0 +1,6 @@
+name: KotlinPluginTemplate
+main: org.simplemc.plugintemplate.KotlinPluginTemplate
+version: 0.1.0
+api-version: "1.15"
+website: https://github.com/SimpleMC/KotlinPluginTemplate
```

Comment:
404 / Not the repositories URL

Might be intentional if you plan on renaming it?

Reply:
naa, bad copy+paste from fixing the main above